js/index.js

```diff
@@ -1,7 +1,133 @@
+Generate 3 paragraphs. Store the text in a variable type of string.
+Make your program count the number of words in the string.
+Make your program count the number of times the Latin word et appears.*/
+
+//Bonus 1.1
+  let loremIpsum = `Lorem ipsum dolor sit amet, consectetur adipiscing elit. In ullamcorper magna condimentum placerat lacinia. Aliquam sed neque suscipit ipsum dapibus vehicula. Fusce consequat, orci at aliquet vulputate, lectus elit interdum arcu, a facilisis libero massa a justo. Phasellus tempus dui odio, ac tincidunt nisi mollis at. Suspendisse varius, nunc sit amet porta suscipit, urna purus dapibus odio, eget interdum nibh ipsum quis risus. Ut lorem dolor, bibendum et lobortis sed, rutrum quis mi. Fusce suscipit porta nisi sit amet molestie. Praesent tortor massa, convallis vitae turpis vulputate, placerat sodales tortor. Aenean dapibus nisi augue, nec tincidunt libero elementum eu. Maecenas consectetur volutpat odio sed fermentum.
+
+Integer tempus lacus eu justo congue rutrum. Etiam in placerat augue. Curabitur rhoncus erat et ante luctus dignissim. Suspendisse et lectus accumsan, tincidunt nunc vitae, semper metus. Cras dapibus sapien nunc, nec mattis est placerat a. Morbi mattis mattis dolor, sed tempor orci. Maecenas vitae nibh mollis purus convallis malesuada. Vestibulum malesuada, metus ac accumsan ultrices, est est tincidunt sem, et volutpat felis velit vel elit. Nam a velit porta, posuere felis sit amet, venenatis purus. Aenean sed ligula pretium, luctus arcu vel, venenatis dolor. Nullam sagittis, lectus non sollicitudin viverra, tellus sapien vehicula sem, sed ultricies urna quam id lacus. Suspendisse iaculis lorem turpis, nec elementum ante hendrerit nec. Pellentesque vestibulum turpis enim. Nam lacus nisl, varius id tincidunt quis, ullamcorper vel orci.
+
+Nullam id justo in lacus condimentum condimentum eu ut tortor. Vivamus tempus, velit at blandit condimentum, ex mi finibus ipsum, et dictum sapien sapien eu est. Vestibulum posuere tortor nunc. Quisque et turpis quis ante commodo luctus. Integer ac enim nunc. Vestibulum eget nulla et diam ultrices tincidunt. Vestibulum fermentum arcu quis consectetur gravida. Curabitur sit amet ligula a dolor iaculis posuere. Sed eu ante sit amet dolor faucibus tempus. Duis semper nulla tellus, et interdum metus pellentesque quis. Praesent dapibus, arcu non egestas elementum, risus mauris imperdiet risus, nec volutpat est ligula in tortor. Curabitur at feugiat felis.`
+
+//Bonus 1.2
+
+var aux =loremIpsum.split("\n")
+// console.log("aux: ",aux, aux.length)
+// var auxTemp = []
+var counter = 0
+for (var i =0;i<aux.length; i++){
+    if (aux[i] != "") {
+        // auxTemp.push(aux[i])
+        var words = aux[i].split(" ")
+        counter = counter + words.length
+    }
+}
+console.log("numero: ",counter)
+// console.log(auxTemp)
+
+//Bonus1.3
+let etCounter = 0;
+
+for ( i = 0; i < loremIpsumArr.length; i++) {
+  if (loremIpsumArr[i] === `et`){
+    etCounter++;
+  }
+}
```

Comment:
Cuidado que esto no funciona. Lo que estáis comparando solo es un caracter. Lo que tendríais que hacer es separarlo por palabras e ir comparando cada una de las palabras si fuesen la palabra `et`.